airbyte-webapp/src/components/base/Tooltip/Tooltip.tsx

```diff
@@ -1,26 +1,32 @@
-import { flip, offset, shift, useFloating } from "@floating-ui/react-dom";
+import { autoUpdate, flip, offset, shift, useFloating, UseFloatingProps } from "@floating-ui/react-dom";
 import classNames from "classnames";
 import React, { useState, useEffect } from "react";
+import { createPortal } from "react-dom";
 
 import { tooltipContext } from "./context";
-import styles from "./ToolTip.module.scss";
-import { ToolTipProps } from "./types";
+import styles from "./Tooltip.module.scss";
+import { TooltipProps } from "./types";
 
 const MOUSE_OUT_TIMEOUT_MS = 50;
 
-export const ToolTip: React.FC<ToolTipProps> = (props) => {
+const FLOATING_OPTIONS: UseFloatingProps = {
+  middleware: [
+    offset(5), // $spacing-sm
+    flip(),
+    shift(),
+  ],
+  whileElementsMounted: autoUpdate,
```

Comment:
Is this needed because you are now using a portal to place the tooltip on the document.body?

Reply:
Yes. Now it needs to move the tooltip when the user scrolls on the page.